ast/term.go

```diff
@@ -154,6 +154,20 @@ func InterfaceToValue(x interface{}) (Value, error) {
 			r.Insert(NewTerm(k), NewTerm(v))
 		}
 		return r, nil
+	case map[interface{}]interface{}:
```

Comment:
We don't need this if [util.Unmarshal](https://godoc.org/github.com/open-policy-agent/opa/util#Unmarshal) is used below.